src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -203,6 +206,38 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
             return new CommandLineSourceRepositoryProvider(SourceProvider);
         }
 
+        private DependencyBehavior TryGetDependencyBehavior(string behaviorStr)
+        {
+            DependencyBehavior dependencyBehavior;
+
+            if (!Enum.TryParse<DependencyBehavior>(behaviorStr, /* ignoreCase */ true, out dependencyBehavior))
+            {
+                throw new CommandLineException(string.Format(CultureInfo.CurrentCulture, LocalizedResourceManager.GetString("InstallCommandUnknownDependencyVersion"), behaviorStr));
+            }
+
+            return dependencyBehavior;
+        }
+
+        private DependencyBehavior GetDependencyBehavior()
+        {
+            // If dependencyVersion is not set by either the config or commandline, default dependency behavior is 'Lowest'.
+            DependencyBehavior dependencyBehavior = DependencyBehavior.Lowest;
+
+            string settingsDependencyVersion = SettingsUtility.GetConfigValue(Settings, "dependencyVersion");
+
+            // Check to see if commandline flag is set. Else check for dependencyVersion in .config.
+            if (!string.IsNullOrEmpty(DependencyVersion))
```

Comment:
I believe this should be the case here. It checks to see if the DependencyVersion (command line) is set. If it is it will use the commandline version. Else it will check for the .config.

I just grab the string from the settings first to be used in the else case.